package.json

```diff
@@ -79,7 +79,8 @@
     "@atjson/source-url": "file:packages/@atjson/source-url"
   },
   "scripts": {
-    "build": "tsc -b packages/**/* --verbose && tsc -b packages/@atjson/**/tsconfig.modules.json --verbose",
+    "build": "tsc -b packages/**/* --verbose && tsc -b packages/@atjson/**/tsconfig.modules.json --verbose && npm run copy-tsd",
+    "copy-tsd": "cp ./packages/@atjson/source-ckeditor/src/ckeditor.d.ts ./packages/@atjson/source-ckeditor/dist/commonjs/ckeditor.d.ts && cp ./packages/@atjson/source-ckeditor/src/ckeditor.d.ts ./packages/@atjson/source-ckeditor/dist/modules/ckeditor.d.ts",
```

Comment:
Sucks that this would potentially need to be expanded with every .d.ts file we add -- i guess since its only the one for now we dont need to over engineer